server/settings.py

```diff
@@ -186,6 +186,7 @@ def env(variable, fallback_value=None):
 MAIL_USERNAME = env('MAIL_USERNAME', 'liveblogsf@gmail.com')
 MAIL_PASSWORD = env('MAIL_PASSWORD', 'fabric2010')
 ADMINS = [MAIL_USERNAME]
+MAIL_FROM = env('MAIL_FROM', 'liveblogsf@gmail.com')
```

Comment:
I'll keep this line in both places because atm our sd repo is not up to date to main sd repo (until we will update everything)